src/components/Navbar.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from 'react';
+import logo from '../images/logo.png';
+import { MenuIcon, XIcon } from '@heroicons/react/outline';
+import SelectLanguage from './SelectLanguage';
+
+function Navbar() {
+  const [dropDownMenu, setUpDropDown] = useState(false);
+
+  const handleClick = (e) => {
+    e.preventDefault();
+    setUpDropDown(!dropDownMenu);
+  };
+
+  return (
+    <div className="w-screen h-[90px] z-10  bg-[#000] bg-opacity-[12%] fixed drop-shadow-lg">
+      <div className="pl-[10%] flex justify-between items-center w-full h-full">
+        <div className="flex items-center w-full justify-between">
+          <img src={logo} alt="/" className="w-36" />
+          <div className="md:flex items-center">
+            <div className="md:pr-[50px]">
+              <ul className="hidden md:flex text-white font-semibold ">
+                <li>Home</li>
+                <li>About</li>
+                <li>Routes</li>
+                <li className="whitespace-nowrap">Contact Us</li>
+              </ul>
+            </div>
+            <div className="hidden md:flex h-[30px] pr-[70px]">
+              <button className="w-[90px] border-button-color border-fon text-white bg-transparent hover:bg-button-color hover:text-white hover:border-none">
+                Sign In
+              </button>
+              <div className="px-10">
+                <SelectLanguage />
+              </div>
```

Comment:
There is no way you can select language on this screen size. Please revisit your responsiveness.

<img width="751" alt="Screen Shot 2022-04-13 at 12 56 58" src="https://user-images.githubusercontent.com/54619678/163166051-575711cb-ea43-4116-bd10-9b8cddab2297.png">

Reply:
please re-review the changes requested has made